Object[] left_goals
Object[] right_goals
